homeassistant/components/blebox/__init__.py

```diff
@@ -0,0 +1,105 @@
+"""The BleBox devices integration."""
+import asyncio
+import logging
+
+from blebox_uniapi.error import Error
+from blebox_uniapi.products import Products
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+CONFIG_SCHEMA = vol.Schema({DOMAIN: vol.Schema({})}, extra=vol.ALLOW_EXTRA)
```

Comment:
Looks like this isn't being used since there is no yaml config.  If you want to allow yaml config you can do something like this
```
async def async_setup(hass: HomeAssistant, config: dict) -> bool:
    """Set up blebox from YAML."""

    hass.data.setdefault(DOMAIN, {})

    if not config.get(DOMAIN):
     return True

    hass.async_create_task(
        hass.config_entries.flow.async_init(
            DOMAIN, context={"source": SOURCE_IMPORT}, data=conf
        )
    )
    return True
```

Reply:
Probably the only config option I'd want to is something like a custom SCAN_INTERVAL or related. (Though this probably only makes sense for testing).

If I can handle that from the config flow, I believe I can do without the YAML config.

(BleBox has very decent and feature-rich app/web UIs to configure their devices - so duplicating this would only unnecessarily increase Home Assistant maintainer work).

So I'll remove it tomorrow (with the last batch of fixes for this PR).

Thanks for putting in so much time and care into this review! I really appreciate it!

Reply:
Ideally as developers we choose a good SCAN_INTERVAL and it shouldn't be something the user has to configure as we should know more about what is going on (hopefully)